lib/srv/desktop/rdp/rdpclient/client.go

```diff
@@ -367,6 +367,7 @@ func (c *Client) startRustRDP(ctx context.Context) error {
 			allow_directory_sharing: C.bool(c.cfg.AllowDirectorySharing),
 			show_desktop_wallpaper:  C.bool(c.cfg.ShowDesktopWallpaper),
 			client_id:               cHostID,
+			keyboard_layout:         C.uint32_t(c.cfg.KeyboardLayout),
```

Comment:
Is the underlying field meant to be a signed int32 or unsigned? I see us converting to unsigned int here, but representing the value as signed in other places.

Reply:
Yeah, I see `int32` everywhere else.
@probakowski Am I alright to convert this, or is there a reason it is this way?

Reply:
The field is defined as uint32 in [RDP spec](https://learn.microsoft.com/en-us/openspecs/windows_protocols/ms-rdpbcgr/00f1da4a-ee9c-421a-852f-c19f92343d73) so I think we should convert everything to unsigned.
Note: it shouldn't matter much as all [defined locales](https://learn.microsoft.com/en-us/previous-versions/windows/it-pro/windows-vista/cc766503(v=ws.10)) are way below max value of int32